modules/service-fips.sh

```diff
@@ -107,11 +109,11 @@ _fips_prep_check() {
             fi
             ;;
     esac
+}
 
-    # check to see if fips pkgs already installed.
-    fips_hmacs="openssh-client-hmac openssh-server-hmac libssl1.0.0-hmac \
-                linux-fips strongswan-hmac"
-    for pkg in $fips_hmacs; do
+_fips_check_packages_installed() {
+    local pkg
+    for pkg in $FIPS_HMAC_PACKAGES; do
         if is_package_installed "$pkg"; then
             if fips_is_enabled; then
```

Comment:
I know this wasn't changed in this branch, but I think this check should be better. It should check that all packages are installed before moving on to fips_is_enabled, because if there is a missing package, we will want to fix it by installing it. And we definitely shouldn't tell the user to reboot if we find out just one package is installed and not the rest.

Reply:
Can you file a separate bug for this?

Reply:
Sure, issue #102